src/commands/install_gcloud.yml

```diff
@@ -0,0 +1,11 @@
+description: >
+  If executor does not have gcloud and kubectl this command install both and add to the PATH
+steps:
+  - run:
+      name: install gcloud
+      command: |
+        curl https://sdk.cloud.google.com > install.sh
+        bash install.sh --disable-prompts
+        export PATH=/home/circleci/google-cloud-sdk/bin/:${PATH}
+        gcloud components install kubectl
+        echo 'export PATH=/home/circleci/google-cloud-sdk/bin/:${PATH}' >> $BASH_ENV
```

Comment:
what is the runtime for this? 😟

Reply:
Hehe yep, I had the same worry... what we agreed with Robert was to first make it work, and then see how to optimize it.

Reply:
my experience 20 months ago was
- pulling a custom executor image adds 1min to your build
- installing gcloud stuff adds >1min to your build

so either way im kinda worried, unless its an image replacing a custom image (in which case the additional time was already a reality)